Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Florian Dehau <work@fdehau.com>"]
 description = """
 A library to build rich terminal user interfaces or dashboards
 """
+documentation = "https://docs.rs/tui/latest/tui"
```

Comment:
Shouldn't the `latest` replaced by the current version of the crate ?

Reply:
Well, yes. But then it needs to be updated every time there is a new release.

Reply:
I'm aware of that but since this is effectively packaged with the release, I assume that it would be better that this link was in sync.

Reply:
Fixed.

Reply:
Thanks